src/Views/UserManagement/AuthUser/register.blade.php

```diff
@@ -0,0 +1,22 @@
+<?php
+if (session_status() === PHP_SESSION_NONE) {
```

Comment:
Unikaj implementowania logiki do szablonów. Jest to bardzo zła praktyka. Logika ta często się potem rozwija. Szablon ma służyć tylko do wyświetlania a nie do obsługiwania jakiejś głębszej logiki.